Cargo.lock

```diff
@@ -8435,10 +8511,10 @@ dependencies = [
 [[package]]
 name = "webpki"
 version = "0.21.0"
-source = "git+https://github.com/scs/webpki-nostd.git#935d31c36fa9b6d55a3226572eaf2b3ded7cf437"
+source = "git+https://github.com/scs/webpki-nostd.git?branch=master#22d1772c39ed9081c2815aadb30e7973f3c4e93f"
 dependencies = [
- "ring 0.16.20",
- "ring 0.16.9",
+ "ring 0.16.20 (registry+https://github.com/rust-lang/crates.io-index)",
+ "ring 0.16.20 (git+https://github.com/Niederb/ring-xous.git?branch=0.16.20-cleanup)",
```

Comment:
Is there a reason that we have both dependencies in the webpki crate?

```
[target.'cfg(target_arch = "wasm32")'.dependencies]
ring_wasmable = { package = "ring", git = "https://github.com/Niederb/ring-xous.git", branch = "0.16.20-cleanup", default-features = false, features = ["alloc"] }

[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
ring = { version = "0.16.20", default-features = false, features = ["alloc"] }
```

@Niederb  Can't the `ring_wasmable` also be used in the normal case?

Reply:
My thinking was to use it only when we really need it, as it is less battle proven than standard ring. I honestly don't know what features we use from ring but depending on the use case (for example cryptography) I would also be a bit concerned about timing attacks. They did some [investigation](https://www.bunniestudios.com/blog/?p=6521) ("But Is it Constant Time?") but I would stick to standard ring unless we have no choice.

Reply:
Alright, thanks for the explanation. I am convinced!